microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -433,20 +433,42 @@ def verify_ibm_mpi(self, environment: Environment, log: Logger) -> None:
         server_ssh.add_known_host(client_ip)
         client_ssh.add_known_host(server_ip)
 
+        if server_node.tools[Ls].path_exists("/usr/share/modules/init/bash"):
```

Comment:
why need to check this bash? what's different if it has or not? If it's really useful, add a comment above.

Reply:
it indicates a tool module has been installed, now I add is_hpc_image field.